docs/tutorials/generate-custom-test-data.rst

```diff
@@ -0,0 +1,120 @@
+.. _ksql_generate-custom-test-data:
+
+Generate custom test data by using the ksql-datagen tool
+========================================================
+
+KSQL enables generating test data that complies with a custom schema
+that you define. 
+
+To generate test data, create an Apache Avro schema and pass it to the
+command-line tool, ``ksql-datagen``, which generates random data according

```

Comment:
we should probably pick either `ksql-datagen command-line tool` or `ksql-datagen tool`.